app/views/shared/_footer.html.erb

```diff
@@ -21,7 +16,6 @@
                     <li><%= link_to "Annoucements", announcements_path %></li>
```

Comment:
chance "Annoucements" to "Announcements"

Reply:
There's a bunch of failing test cases as well, if you know what the cause is from your changes please fix them otherwise I can try looking at them